### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by glenngnng**
Change images
Moved images into respective folders
Add or remove spaces for formatting

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -229,7 +231,7 @@ A person can have any number of tags (including 0)
 * For example, for Robert Johnson (the example command), it would be:<br> `New person added: Robert Johnson; Phone: 55512345; Email: robertj@email.com; Occupation: Hairdresser; Address: 789 Oak Street, Suite 10; AppointmentDate: ; Tags: `<br> Please note that both Appointment Date and Tags are empty as they are not necessary for adding a person
```
Comment (bhnuka) [inline docs/UserGuide.md:231]: for some reason, it says Robert Johnson (the example command) when it wasnt mentioned prior. Should change the phrasing or add Robert Johnson to the example commands.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -441,7 +442,7 @@ Edits an existing client's parameter in FAPro.
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags
 
-![edit format](images/edit-UG/editformat.png)
+  ![edit format](images/edit-UG/editformat.png)
```
Comment (bhnuka) [inline docs/UserGuide.md:445]: For some reason this screenshot is really low resolution

**Review by bhnuka — COMMENTED** [review]
paiseh glenn... need to update a few more screenshots...

add the following screenshots:
help window
would be good to add a calendar_success (like questionaire and help)

should crop the following screenshots:
editresult
editfailure
find_n2
find_a2
find_appt2
find_appt3
find_appt4
maybe can remove the single delete example for undo? since we have multiple delete

one small change in text (shown in other comments below)

and if possible update editformat coz its low res

thanks :-)

**Comment by glenngnng** [comment]
Alrighty, I'll amend all of the above comments in another PR! Thanks Bhanuka!

**Pull request merged**